src/registry_schemas/schemas/party.json

```
@@ -73,7 +73,8 @@
                         "Incorporator",
                         "Proprietor",
                         "Partner",
-                        "Applicant"
+                        "Applicant",
+                        "Appoint Receiver"
```

Role should be receiver. `Appoint Receiver` is an action (filing) not a role
```suggestion
                        "Receiver"
```

---

Updated